README.md

```diff
@@ -269,6 +270,30 @@ You can run all the unit tests using nosetests. Most of the tests are mocked.
 $ nosetests --verbosity=2 tests/
 ```
 
+
+## Local Development
+
+There are many ways to authenticate with Cerberus and they don't always work on local. Luckily you can easily get a token from the dashboard and export it to your environment variables. Doing so will override the authentication mechanism that's set in place. For example:
```

Comment:
I recommend wording this a little more formally. For example:

```The easiest way to locally test the Python client is to first authenticate with Cerberus through the dashboard, then use your dashboard authentication token to make subsequent calls. See examples below.```

Reply:
Copy pasted, also updated one other sentence